config/set/php74.php

```diff
@@ -43,6 +42,5 @@
         CurlyToSquareBracketArrayStringRector::class,
         MoneyFormatToNumberFormatRector::class,
         ParenthesizeNestedTernaryRector::class,
-        TypedPropertyFromAssignsRector::class,
```

Comment:
Instead of hiding this big changed in PHP 7.4 set, the whole type declarations should be applied instead.
It's more relialble and can fill typed properties from various sources.